### PR TITLE
Create image for Amazon Linux

### PR DESCRIPTION
**Pull request by tjoelsson**
## Summary of changes

- Create image for Amazon Linux working with Vagrant
- Add PR template

## How to Test

1. Make sure you have latest Docker installed
1. Run `$ docker build -t juwaicom/amazonlinux-vagrant:1.0 amazonlinux-vagrant/`

### CODE REVIEW

File: amazonlinux-vagrant/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+# Vagrant user
+ADD files/insecure_key /tmp/insecure_key
+RUN adduser vagrant \
+    && usermod -aG wheel vagrant \
+    && sudo -u vagrant mkdir -p /home/vagrant/.ssh \
+    && chmod 700 /home/vagrant/.ssh \
+    && sudo -u vagrant touch /home/vagrant/.ssh/authorized_keys \
+    && chmod 600 /home/vagrant/.ssh/authorized_keys \
+    && cat /tmp/insecure_key >> /home/vagrant/.ssh/authorized_keys \
```
Comment (mark-juwai) [inline amazonlinux-vagrant/Dockerfile:49]: ? Remove `/tmp/insecure_key`?

**Review by mark-juwai — DISMISSED** [review]

File: amazonlinux-vagrant/Dockerfile (hunk 1 above)
Comment (tjoelsson) [reply amazonlinux-vagrant/Dockerfile:49]: Updated

**Review by mark-juwai — APPROVED** [review]

**Comment by mark-juwai** [comment]
Close this for now, it's the only one in my PR list. :)

**Comment by tjoelsson** [comment]
Or... someone can approve it? 😄

**Review by zhangjustin — APPROVED** [review]

**Pull request merged**